plugins/renamerOnUpdate/renamerOnUpdate.py

```diff
@@ -450,7 +466,12 @@ def extract_info(scene: dict, template: None):
     scene_information['current_directory'] = os.path.dirname(scene_information['current_path'])
     scene_information['oshash'] = scene['oshash']
     scene_information['checksum'] = scene.get("checksum")
+    scene_information['studio_code'] = scene.get("code")
 
+    if scene.get("stash_ids"):
+        #todo support other db that stashdb ?
+        scene_information['stashid_scene'] = scene['stash_ids'][0]["stash_id"]
```

Comment:
Haven't actually tested yet
If the scene doesnt have any stash_ids will the above work ok?
Won't an empty array pass the `if` and then fail due to `scene['stash_ids'][0]` ?